releasecalendar/client_test.go

```diff
@@ -177,7 +189,7 @@ func TestGetLegacyRelease(t *testing.T) {
 		})
 	})
 
-	Convey("Given that 200 OK is returned by the API with an empty body", t, func() {
+	Convey("Given that 500 StatusInternalServerError is returned by the API with an empty body", t, func() {
```

Comment:
Is this change correct? It looks like the test client does return 200 OK?

Reply:
let me have a look

Reply:
as below